app/controllers/tasks_controller.rb

```diff
@@ -3,54 +3,79 @@
 class TasksController < ApplicationController
   before_action :authenticate_user!
 
-  before_action :set_task, only: [:show, :edit, :update, :destroy]
+  before_action :set_task, only: [:show, :edit, :update, :destroy, :inform]
   before_action :set_job, only: [:index]
 
   respond_to :html
 
+  # GET /job/tasks?job_id=1
   def index
     @tasks = @job.tasks
-    respond_with(@tasks)
   end
 
+  # GET /tasks/1
   def show
-    respond_with(@task)
   end
 
+  # GET /tasks/new
   def new
     @task = Task.new
-    respond_with(@task)
   end
 
+  # GET /tasks/1/edit
   def edit
   end
 
+  # POST /tasks
   def create
     @task = Task.new(task_params)
-    @task.save
-    respond_with(@task)
+    if @task.save
+      redirect_to @task, notice: 'Task was successfully created.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.